src/include/iscsi-lio-server/TargetData.rb

```diff
@@ -301,11 +299,10 @@ def fetch_target(target_name)
   def get_keys
     test = nil
     @target_hash_list.each do |key, value|
-       test = key
+      test = key
```

Comment:
style ways nice change, but code wise. WTF? This method will return the last key in `@target_hash_list`, this looks very strange and definitively not what you expect from method name. Can you please add at least FIXME ? As this looks really strange.

Reply:
Added comment.